string private_key_path
string token_address
string did
float32 tokens_nomber
float32 ocean_amount